react.md

```diff
@@ -890,12 +890,53 @@ Now your test is a lot simpler and as a happy side effect you can easily re-use
 ### Integration/Functional Testing
 [Cypress](https://www.cypress.io/)
 
-## Linting Tools
-Linting is *mandatory*
-- TODO: Document this
+## Linting & Coding Style
+We've adopted a standard coding style in order to make our projects easier to onboard and cut down on worries about tabs vs. spaces, casing styles, etc. This coding standard will be documented in brief here, but is fully captured in our [`eslint-config-domain7` NPM package](https://github.com/domain7/eslint-config-domain7). It's *highly recommended* to configure your text editor to automatically lint your React code as you work, as our React projects often have pre-commit & pre-push checks that will fail if your code doesn't pass. It's much easier to learn and adapt a standard when you get feedback in small chunks instead of all at once.
+
+Our standard extends [eslint-config-airbnb](https://github.com/airbnb/javascript). Please review their documentation (and the React sub-page.)
+
+### Our customizations
+* We automatically include `jest` as a global and bundle the `react`, `jsx-a11y`, and `filenames` plugins.
+* We allow non-destructuring props assignment in case destructuring would cause name conflicts.
+* We allow dangling underscores (i.e. `const variable__`)
+* We enforce dangling commas in multi-line array, object, and function arguments lists. This makes it easier to re-order lines without causing syntax errors. This is not enforced on single-line lists.
```

Comment:
`... makes it easier to re-order lines without causing syntax errors` 
Thanks for including this point - it's why I like it, but it doesn't' a  lot of "why we lint" guides explicitly